openlattice.yaml

```diff
@@ -1533,7 +1533,29 @@ paths:
   ## DATA APIs ##
   ###############
 
-  /datastore/data/associations/:
+  /datastore/data:
+    post:
+      summary: Create entity and assocation 
```

Comment:
minor, but I feel like this description is misleading -- I'd prefer either `Create entity and association data` or `Create entities and associations` so that there's no implication that this only operates on a single entity/association